json/jsonparse.cc

```diff
@@ -732,6 +747,18 @@ void json_import(Context *ctx, string modname, JsonNode *node)
 
             here = cell_parent->data_dict.at(cell_parent->data_dict_keys[nnid]);
             std::string basename = cell_parent->data_dict_keys[nnid];
+            bool upto = false;
+            int start_offset = 0;
+            if (here->data_dict.count("upto") != 0) {
+				JsonNode *val = here->data_dict.at("upto");
+				if (val->type == 'N')
+					upto = val->data_number != 0;
+			}
```

Comment:
This indenting doesn't look right..  ;)

Reply:
Fixed now. Thanks for pointing out to this.